recipes/automake/all/test_package/conanfile.py

```diff
@@ -1,89 +1,29 @@
-from conans import AutoToolsBuildEnvironment, ConanFile, tools
-from conan.tools.microsoft import is_msvc
-from contextlib import contextmanager
-import os
-import shutil
+from os import environ
 
-required_conan_version = ">=1.45.0"
+from conan import ConanFile
+from conan.tools.build import can_run
+
+required_conan_version = ">=1.50.0"
 
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    exports_sources = "configure.ac", "Makefile.am", "test_package_1.c", "test_package.cpp"
-    # DON'T COPY extra.m4 TO BUILD FOLDER!!!
+    generators = "VirtualBuildEnv"
     test_type = "explicit"
+    win_bash = True
```

Comment:
to remove when msys2 fixed

Reply:
Will add a fixme comment here, is there an issue nummer which I can reference?